addon/mixins/has-behaviors.js

```diff
@@ -0,0 +1,270 @@
+import Ember from 'ember';
+import withBackingField from 'ember-light-table/utils/with-backing-field';
+import ekbListenerName from 'ember-keyboard/utils/listener-name';
+
+const { A, observer, on } = Ember;
+
+/*
+ * from ember-keyboard
+ */
+const gatherKeys = function gatherKeys(event) {
```

Comment:
The `gatherKeys` function is not part of their public API but I wanted to use the ember-keyboard syntax for row click event names (rowClick:shift, rowClick:shift+ctrl, etc.), to keep things consistent. So I just copied the method that gather the modifiers keys from the event.